lib/beaker-hostgenerator/cli.rb

```diff
@@ -54,6 +54,16 @@ def initialize(argv = ARGV.dup)
    1 CentOS 6 64 bit node with roles = master, hypervisor = none, node name = static1, and my-key = my-value
    1 CentOS 6 32 bit node with roles = agent and the default hypervisor
 
+example with arbitrary host settings as a list:
```

Comment:
Nitpicky but this kind of sounds like an alternate way to specify the arbitrary settings when we say "arbitrary host settings __as__ a list".

Maybe try rewording this so it's clear the word "list" is referring to the setting and not the outer map structure.